plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -986,30 +987,35 @@ func (c *PodAffinityChecker) InterPodAffinityMatches(pod *v1.Pod, meta interface
 // First return value indicates whether a matching pod exists on a node that matches the topology key,
 // while the second return value indicates whether a matching pod exists anywhere.
 // TODO: Do we really need any pod matching, or all pods matching? I think the latter.
-func (c *PodAffinityChecker) anyPodMatchesPodAffinityTerm(pod *v1.Pod, allPods []*v1.Pod, node *v1.Node, term *v1.PodAffinityTerm) (bool, bool, error) {
+func (c *PodAffinityChecker) anyPodMatchesPodAffinityTerm(pod *v1.Pod, allPods []*v1.Pod, node *v1.Node, term *v1.PodAffinityTerm) (bool, bool, []error) {
 	if len(term.TopologyKey) == 0 {
-		return false, false, errors.New("Empty topologyKey is not allowed except for PreferredDuringScheduling pod anti-affinity")
+		return false, false, []error{errors.New("Empty topologyKey is not allowed except for PreferredDuringScheduling pod anti-affinity")}
 	}
-	matchingPodExists := false
+
 	namespaces := priorityutil.GetNamespacesFromPodAffinityTerm(pod, term)
 	selector, err := metav1.LabelSelectorAsSelector(term.LabelSelector)
 	if err != nil {
-		return false, false, err
+		return false, false, []error{err}
 	}
+
+	matchingPodAnywhere := int32(1)
```

Comment:
Yes, I changed from 0 to 1 :). 0 makes it means how many OTHER pods that matching the term; so I think 1 is better: how many matching pods in total.